src/ui/markdown-components.tsx

```diff
@@ -131,6 +131,8 @@ export default {
 			return (
 				<A
 					class="[&>code]:shadow-[0_0_0_1px_#38bdf8] hover:[&>code]:shadow-[0_0_0_2px_#38bdf8]"
+					target="_blank"
+					rel="noopener noreferrer"
```

Comment:
I believe this will make even internal links open in a new tab.
Can we perhaps only add them if the URL is absolute?